Packs/RiskSense/Playbooks/BlueKeep_CVE_Exposure_-_RiskSense.yml

```diff
@@ -0,0 +1,460 @@
+id: BlueKeep CVE Exposure - RiskSense
+version: -1
+name: BlueKeep CVE Exposure - RiskSense
+description: Block IPs and apply the tag to assets that are vulnerable to BlueKeep CVE
+  (CVE-2019-0708).
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 296e7f69-00e3-498b-8ddc-d556d5e7f0a0
+    type: start
+    task:
+      id: 296e7f69-00e3-498b-8ddc-d556d5e7f0a0
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: f9a5a481-c591-436f-8e11-4ac6518c0231
+    type: condition
+    task:
+      id: f9a5a481-c591-436f-8e11-4ac6518c0231
+      version: -1
+      name: Is RiskSense enabled ?
+      description: Returns 'yes' if integration brand is available. Otherwise returns
+        'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "10"
+      "yes":
+      - "13"
+    scriptarguments:
+      brandname:
+        simple: RiskSense
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
```

Comment:
This is a deprecated command
Use this sub playbook instead
Block IP - Generic v2